crates/rpc-client/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "op-alloy-rpc-client"
+description = "Optimism RPC Client"
+
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+homepage.workspace = true
+authors.workspace = true
+repository.workspace = true
+exclude.workspace = true
+
+[dependencies]
+# Alloy
+op-alloy-rpc-types.workspace = true
+alloy-eips.workspace = true
+
+# rpc
+jsonrpsee.workspace = true
+jsonrpsee-types.workspace = true
+
+[features]
+client = [
+    "jsonrpsee/client",
+    "jsonrpsee/async-client",
+]
```

Comment:
hmm by default this now generates jsonrpsee server traits, so perhaps `rpc-client` isn't the perfect fit here.